src/database/DataTypes/ItemStoreJoin.js

```diff
@@ -15,21 +15,14 @@ import Realm from 'realm';
 
 export class ItemStoreJoin extends Realm.Object {
   /**
-   * Delete item to store join and update visibility of items in current store.
-   *
+   * Make the related item to this join invisible on deletion.
    * @param  {Realm}  database
    */
   destructor(database) {
-    // Check if join is associated with this store. If not, nothing to be done.
     if (!this.joinsThisStore) return;
-
-    // Check if join is associated with item in this database. If not, nothing
-    // to be done.
     const itemResults = database.objects('Item').filtered('id == $0', this.itemId);
     if (!itemResults || itemResults.length <= 0) return;
 
-    // Join is associate with this store and item exists in this database. update
-    // item to no longer be visible in this store.
```

Comment:
for some reason these comments made me angry enough to delete them

Reply:
😕 The comments don't add anything, better to remove!